src/main/java/com/google/firebase/auth/FirebaseAuth.java

```diff
@@ -41,7 +40,7 @@
  * custom tokens for use by client-side code, verifying Firebase ID Tokens received from clients, or
  * creating new FirebaseApp instances that are scoped to a particular authentication UID.
  */
-public class FirebaseAuth extends AbstractFirebaseAuth {
+public final class FirebaseAuth extends AbstractFirebaseAuth {
```

Comment:
Should we make `TenantAwareFirebaseAuth` final for consistency?

Reply:
I think we should. Lets do that in a separate PR to keep this merge commit clean.

Reply:
Alright! I'll slip that into `tenant-mgt` in a separate commit.